Vagrantfile

```diff
@@ -10,13 +10,14 @@ Vagrant.configure("2") do |config|
     ## 4 to 8 CPUs seems to be about the minimum for e2e tests
     # domain.cpus = 8
     domain.machine_virtual_size = 40
-  end
 
-  config.vm.provision "shell", inline: <<-SHELL
+    config.vm.provision "shell", inline: <<-SHELL
```

Comment:
is this an intentional indentation change?

Reply:
yeah, it's moving the image resize just into the libvirt setup